src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMeshQueueMultiMessageTest.java

```diff
@@ -94,7 +94,10 @@ private void assertGpOutboundQueueMessages(SoftAssertions softly, Message messag
                 new Customization("identifier.value", IGNORE),
                 new Customization("entry[*].fullUrl", IGNORE),
                 new Customization("entry[*].resource.**.reference", IGNORE),
-                new Customization("entry[*].resource.id", IGNORE)
+                new Customization("entry[*].resource.id", IGNORE),
+                new Customization("entry[*].resource.subject.reference", IGNORE),
+                new Customization("entry[*].resource.id", IGNORE),
```

Comment:
Looks like 

`new Customization("entry[*].resource.id", IGNORE),`

may be duplicated.

Reply:
Removed in 3f054a9